content/02.body.md

```diff
@@ -3,45 +3,45 @@
 
 Inspection bias indicates that entities are not uniformly studied [@doi:10.1038/nature04209], and it is likely to cause degree bias when networks are constructed using hypothesis-driven findings extracted from the literature, as newly-discovered relationships are not randomly sampled from the set of all true relationships.
 Though there is a high correlation between the number of publications mentioning a gene and its degree in low-throughput interaction networks, the number of publications mentioning a gene has little correlation with its degree in a systematically-derived protein interaction network [Figure 6A in @doi:10.1016/j.cell.2014.10.050].
 This evidence suggests that many poorly studied genes have similar numbers of interactions as those scientists have preferentially examined and that these edges are missed due to inspection bias.
-For networks with strong inspection bias, reliance on degree can lead to predictions that have good metrics when assessed by cross validation but little ability to generalize.
+For networks with a large inspection bias, reliance on degree can lead to predictions that have good metrics when assessed by cross validation but little ability to generalize.
 
-Another reason why a method's reliance on degree can be unfavorable is that degree imbalance can lead to prediction nonspecificity.
+Another reason why a reliance on degree can be unfavorable is that degree imbalance can lead to prediction nonspecificity.
 Nonspecific predictions are not made on the basis of the specific connectivity information contained in a network.
 For example, Gillis et al. [@doi:10.1371/journal.pone.0017258] examined the concept of prediction specificity in the context of gene function prediction and found that many predictions appear to rely primarily on multifunctionality and could be "potentially misleading with respect to causality."
-Real networks have a variety of degree distributions (Figure {@fig:hetionet}), and they commonly exhibit degree imbalance [@doi:10.1371/journal.pone.0017645; @doi:10.1007/978-1-61779-361-5_13; @doi:10.1038/s41467-019-08746-5; @doi:10.1126/science.286.5439.509].
 Degree imbalance leads high-degree nodes to dominate in the predictions made by degree-associated methods [@doi:10.1093/bioinformatics/btv215], which are effective predictors of connections in some biological networks [@doi:10.1186/1752-0509-2-11].
 Consequently, degree-based predictions are more likely nonspecific, meaning the same set of predictions performs well for different tasks.
 
 Depending on the prediction task, edge predictions involving very high degree nodes may be undesired, uninsightful, or nonspecific.
+While nonspecificity may be acceptable in some cases, we believe that specific predictions are desirable for a range of prediction tasks in biomedicine as they represent insight gained from the actual connections in a network.
```

Comment:
I'm not sure I love this wording. I think what you're going for is that you're more interested in predictions that come from the nodes that are connected. Even what you're calling non-specific here is driven by the actual connections in the network (degree is a function of this).

Reply:
```suggestion
While nonspecific predictions may be acceptable in some cases, we believe that specific predictions are desirable for a range of prediction tasks in biomedicine as they represent insight gained from the particular connections in a network.
```
I'm trying to explain what I mean by specificity without using the word 'specific'. Would this be more clear?

Reply:
I think for the latter part you want it to be about the nodes that are connected by edges. I _think_ that's what you mean.

Reply:
That is what I mean. Predictions should reflect the edges between nodes (though obviously we are not predicting connections between directly connected egdes).
```suggestion
While nonspecificity may be acceptable in some cases, we believe that predictions based on the edges and network structure connecting two nodes are desirable for a range of prediction tasks in biomedicine.
```

Reply:
I think all the potential phrasings are okay. I see why it's a hard concept to communicate succinctly. Feel free to use any of the following for inspiration.

```suggestion
While predictions based on primarily on degree may be acceptable for some tasks,
generating less obvious insights from networks requires drawing inferences from particular connections, where analysis considers both the source and target node of an edge.
```

Reply:
I like the suggestion! How about merging the two phrasings like this?
```suggestion
While predictions based primarily on degree may be acceptable for some tasks, generating less obvious insights from networks requires drawing inferences from the specific connections and network structure between nodes.
```

Reply:
> How about merging the two phrasings like this?

looks good to me